tests/codegen/sve/application_axpy_test.py

```diff
@@ -20,6 +21,7 @@ def multiplication(i):
         out = in_A * in_X + in_Y
 
 
+@pytest.mark.skip
```

Comment:
skip? Why not `@pytest.mark.sve` and add it to the markers in `pytest.ini` (as well as disable it by default in `.github/workflows`)